systemd/systemd.go

```diff
@@ -692,12 +705,56 @@ func IsTimeout(err error) bool {
 	return isTimeout
 }
 
+func (l Log) parseLogRawMessageString(key string, sliceHandler func([]string) (string, error)) (string, error) {
+	valObject, ok := l[key]
+	if !ok || valObject == nil {
+		// NOTE: journalctl says that sometimes if a json string would be too
+		// large null is returned, so we may miss a message here
+		return "", fmt.Errorf("key %q missing from message", key)
```

Comment:
Would it make sense to indicate such messages somehow in our output, e.g. "[message truncated]" or some such?

Reply:
yes perhaps we could distinguish between the `!ok` case and the `!valObject` case so we could return different errors

Reply:
message is now different for the `!ok` vs `!valObject` cases